release-team/role-handbooks/emeritus-adviser/README.md

```diff
@@ -40,6 +40,11 @@ Weeks: -2 to 0
 
 At least 2 weeks before the new release cycle begins, the EA is responsible for posting the "Assemble the X.Y Release Team" issue in the SIG-Release repo ([example](https://github.com/kubernetes/sig-release/issues/776)).  They should then make sure that the new Release Lead is officially nominated and chosen, and then issue a call for nominations/volunteers for the other Role Leads.
 
+### Lead Onboarding
+
+1. Ensure the outgoing EA or one of the SIG Release co-chairs has added you in as an owner of the [Kubernetes Release Calendar](https://calendar.google.com/calendar/embed?src=agst.us_b07popf7t4avmt4km7eq5tk5ao%40group.calendar.google.com) Google calendar
```

Comment:
```suggestion
1. Ensure the outgoing EA or one of the SIG Release co-chairs has added you in as an owner of the [Kubernetes Release Calendar](https://bit.ly/k8s-release-cal) Google calendar
```

Reply:
I changed the link to the bit.ly short link. Thank you for catching that.